src/Services/LDAPService.php

```diff
@@ -569,7 +569,11 @@ public function updateMemberFromLDAP(Member $member, $data = null, $updateGroups
                 continue;
             }
 
-            if ($attribute == 'thumbnailphoto') {
+            $photoAttributes = ['thumbnailphoto'];
+
+            $this->extend('updatePhotoAttributes', $photoAttributes);
```

Comment:
I feel like you could make this a config array instead of an extension point. Thoughts?